service/amazee/amazee.go

```diff
@@ -33,7 +34,7 @@ func list() ([]types.ImageSummary, error) {
 func pull_all() {
 	list, _ := list()
 	for _, image := range list {
-		if strings.Contains(fmt.Sprint(image.RepoTags), "amazeeio") || strings.Contains(fmt.Sprint(image.RepoTags), "mailhog/mailhog") || strings.Contains(fmt.Sprint(image.RepoTags), "andyshinn/dnsmasq") {
+		if strings.Contains(fmt.Sprint(image.RepoTags), "amazeeio/") || strings.Contains(fmt.Sprint(image.RepoTags), "mailhog/mailhog") || strings.Contains(fmt.Sprint(image.RepoTags), "andyshinn/dnsmasq") {
```

Comment:
I fixed up this with `amazeeio/` because it was trying to pull anything that started with `amazeeio` in my list, which is a bunch of containers that may or may not exist like `amazeeiolagoon/<imagex>`

Reply:
Very good point, thanks for the explanation.